packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -174,10 +179,15 @@ export const selectDevice = (device?: Device | TrezorDevice) => async (
     });
 };
 
-export const rememberDevice = (payload: TrezorDevice): Action => ({
+export const rememberDevice = (
+    payload: TrezorDevice,
+    forceRemember: true | undefined = undefined,
```

Comment:
```suggestion
    forceRemember?: true,
```
it will be undefined by default

Reply:
It is a function parameter: "Parameter cannot have question mark and initializer.ts(1015)"

Reply:
do not use initializer ( = undefined) just `forceRemember?: true`
it will be "true | undefined" and undefined by defualt